tests/unit_tests/test_nwbfile_metadata.py

```diff
@@ -56,6 +59,30 @@ def test_check_session_start_time_future_date_pass():
     assert check_session_start_time_future_date(nwbfile) is None
 
 
+@pytest.mark.skipif(pynwb.__version__ < "2.7.0", reason="Feature not supported in pynwb < 2.7.0")
+def test_check_session_start_time_contains_time_zone_pass():
+    nwbfile = NWBFile(
+        session_description="",
+        identifier=str(uuid4()),
+        session_start_time=datetime(2010, 1, 1, 0, 0, 0, 0, timezone.utc),
+    )
+    assert check_session_start_time_contains_time_zone(nwbfile) is None
+
+
+@pytest.mark.skipif(pynwb.__version__ < "2.7.0", reason="Feature not supported in pynwb < 2.7.0")
+def test_check_session_start_time_contains_time_zone_fail():
+    session_start_time = datetime(2010, 1, 1, 0, 0, 0, 0)
```

Comment:
Should there be a skip for it it's a `date` instead of `datetime`? If hourly precision isn't even known, I don't think the timezone matters a whole lot

Reply:
I don't think dates can contain timezones